### DISCUSSION TITLE
Does juicefs support Lustre® filesystem?

### DISCUSSION THREAD

**Discussion by jiangke-sr** [juicedata/juicefs#4078] [General]

**Comment by SandyXSD** [comment]
Not sure what you were asking. Please check the [architecture of JuiceFS](https://github.com/juicedata/juicefs#architecture) first and describe your question in more detail.

**Comment by davies** [comment]
@aakejiang Do you mean use Lustre as the data store for JuiceFS? It could be supported via file:// or nfs:// (#3823).

**Reply by jiangke-sr** [reply]
@davies exactly, i'm using Lustre as the data store.

**Reply by flipfloptech** [reply]
Personally, I don't see why you would. My experience with lustre via DDN has been subpar. The only performance gains I see between my lustre deployment and juicefs/tikv/minio deployment is mostly due to SSD, 60 more disks, and infiniband. I've made jokes with my team and DDN themselves about ripping lustre off of the SFA and just deploying minio/tikv/juicefs across it instead. 

Lustre is seemingly only remotely good for extremely large files, where my JFS/MINIO/TiKV combo seems to perform great for everything else.
